Documentation/driver-api/index.rst

```diff
@@ -101,6 +101,7 @@ available subsections can be seen below.
    sync_file
    vfio-mediated-device
    vfio
+   virtual_bus
```

Comment:
comment on commit message:

This bus is also meant to be generic and be available
to be used by any type of device (PCI/ACPI/DT).

-> any type of parent device (PCI/ACPI/DT)?